admin_guide/manage_users.adoc

```diff
@@ -0,0 +1,135 @@
+= Managing Users
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Adding Users
```

Comment:
Remember to add manual, unique IDs for headings (for example, check to make sure there's not some other heading in the repo named "adding-users").